config/webpack.common.js

```diff
@@ -155,6 +155,11 @@ module.exports = (prodMode) =>
                 filename: 'heartbeat.html',
                 template: './views/heartbeat.html',
                 chunks: ['polyfills', 'vendor', 'heartbeat'],
-            })
+            }),
+            new HtmlWebpackPlugin({
+                filename: 'tutorial.html',
+                template: './views/tutorial.html',
+            }),
+            new RedirectPlugin({})
```

Comment:
If you're adding the tutorial here, you should also remove it from above:

               {
                    from: './views/tutorial.html',
                    to: 'tutorial.html',
                },